docs/nada-lang-operators.md

```diff
@@ -0,0 +1,38 @@
+# Built-In Operations
+
+Overview of the primitive and the array Nada operations.
+
+
+## Primitive Operations
+
+| Name            | Syntax                | Supported Types (`P: Public`, `S: Secret`) |
+| --------------- | --------------------- | --------------------- |
+| Addition        | `x + y`         | `P + P`, `P + S`, `S + P`, `S + S` |
+| Subtraction     | `x - y` | `P - P`, `P - S`, `S - P`, `S - S` |
+| Multiplication  | `x * y`            | `P * P`, `P * S`, `S * P`, `S * S`|
+| Power  | `x ** y`            | `P ** P` |
+| Division  | `x / y`            | `P / P`, `P / S`, `S / P`, `S / S` |
+| Modulo  | `x % y`            | `P % P`, `P % S`, `S % P`, `S % S` |
+| Comparisons     | `x < y`, `x <= y`, `x > y`, `x >= y` | `P < P`, `P < S`, `S < P`, `S < S` |
+| Ternary "if else" with public condition    | `cond.if_else(x, y)` | `P.if_else(P, P)`, `P.if_else(P, S)`, `P.if_else(S, P)`, `P.if_else(S, S)` |
```

Comment:
I miss a comment on the output type for all operations, but especially for IfElse.

Reply:
done